modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/jdbc/JdbcRequestHandler.java

```diff
@@ -723,6 +745,21 @@ private JdbcResponse executeQuery(JdbcQueryExecuteRequest req) {
         }
     }
 
+    /** */
+    private List<FieldsQueryCursor<List<?>>> querySqlFields(SqlFieldsQueryEx qry, GridQueryCancel cancel) {
+        if (experimentalQueryEngine != null) {
+            try {
+                return experimentalQueryEngine.query(QueryContext.of(qry, cancel), qry.getSchema(), qry.getSql(), qry.getArgs());
+            }
+            catch (IgniteSQLException e) {
+                U.warn(log, "Failed to execute SQL query using experimental engine. [qry=" + qry + ']', e);
```

Comment:
I think we should not write this warning to log for DDL/DML queries.

Reply:
Not sure, actually here we highlight an issue - it's been handled but still an issue and a user should be aware on it, because this mean you were wrong setting "useExperimentalEngine" flag to true.